lib/events/filelog.go

```diff
@@ -517,21 +551,12 @@ func (l *FileLog) findInFile(fn string, eventFilter []string, total *int, limit
 			}
 		}
 
-		id := ef.GetString(EventID)
-		if id == startKey {
-			foundStart = true
-		}
-
-		if (accepted || !doFilter) && foundStart {
+		if accepted || !doFilter {
```

Comment:
nit: can we get rid of `doFilter` flag by setting accepted to true if `len(eventFilter) == 0 `

Reply:
Fixed